plugins/dependencies/src/main/kotlin/dependencies/Google.kt

```diff
@@ -595,6 +596,50 @@ object Google : IsNotADependency {
         }
     }
 
+    val horologist = Horologist
+
+    /**
+     * Horologist is a group of libraries that aim to supplement
+     * Wear OS developers with features that are commonly required
+     * by developers but not yet available.
+     *
+     * - [Official website here](https://google.github.io/horologist/)
+     * - GitHub page: https://github.com/google/horologist
+     */
+    @Incubating
+    object Horologist : DependencyGroup(group = "com.google.android.horologist") {
```

Comment:
Missing a rule, see file dependencies-versions-key-validated.txt

Shouuld be something like this:

```
        rawRules = """
            com.google.android.horologist:*
                ^^^^^^^        ^^^^^^^^^^
        """.trimIndent()
```